inspirehep/modules/records/receivers.py

```diff
@@ -103,32 +100,6 @@ def populate_inspire_document_type(sender, json, *args, **kwargs):
     json['facet_inspire_doc_type'] = result
 
 
-def match_valid_experiments(sender, json, *args, **kwargs):
-    """Normalize the experiment names before indexing.
-
-    FIXME: this is currently using a static Python dictionary, while it should
-    use the current dynamic state of the Experiments collection.
-    """
-    def _normalize(experiment):
-        try:
-            result = EXPERIMENTS_MAP[experiment.lower().replace(' ', '')]
-        except KeyError:
-            result = experiment
-
-        return result
-
-    if 'accelerator_experiments' in json:
-        accelerator_exps = json['accelerator_experiments']
-        for accelerator_exp in accelerator_exps:
-            facet_experiment = []
-            if 'experiment' in accelerator_exp:
-                experiments = force_list(accelerator_exp['experiment'])
-                for experiment in experiments:
-                    normalized_experiment = _normalize(experiment)
-                    facet_experiment.append(normalized_experiment)
-                accelerator_exp['facet_experiment'] = [facet_experiment]
```

Comment:
This means two things:
1. We need to update `inspire-search-js`'s facet to point to the right thing
2. This normalization has to happen once and for all in DoJSON, not unlike https://github.com/inspirehep/inspire-next/pull/2665.

Reply:
No, actually the mapping was already doing the right thing, minus the normalization: https://github.com/inspirehep/inspire-next/blob/44b72cbad65140c34414e0dfbb3472d81d673ff8/inspirehep/modules/records/mappings/records/hep.json#L136-L139.